docs/lib/datastore/fragments/ios/getting-started/40_codegen.md

```diff
@@ -0,0 +1,14 @@
+If you have chosen you use the Amplify/Tools pod, we will now use it to generate the models from the schema defined in `schema.graphql` and automatically add the new files to the project. If you have chosen to use the Amplify CLI instead, please skip this section, and follow the [Code generation: Amplify CLI](#code-generation-amplify-cli) section.
+
+1. **Update the amplifytools.xcconfig file** and change `modelgen=false` to:
```

Comment:
Do we need to also show the developer, or mention, that they can also set their own profile? Also, what is the default profile here if none is chosen?

Reply:
there's a problem customizing the profile right now since `amplify-app` initializes the backend in headless mode using the `default` profile. My understanding is that in order to change the profile you have to do `amplify init` yourself and pick a different profile.

Reply:
What if there is no profile named `default`?

Reply:
that is a known bug in `amplify-app` I believe, let me try to reference the issue.